graphs/eacs/src/test/kotlin/com/anaplan/engineering/azuki/graphs/eacs/BEH8.kt

```diff
@@ -0,0 +1,67 @@
+package com.anaplan.engineering.azuki.graphs.eacs
+
+import com.anaplan.engineering.azuki.core.runner.Eac
+import com.anaplan.engineering.azuki.core.system.BEH
+import com.anaplan.engineering.azuki.graphs.adapter.api.GraphBehaviours
+import com.anaplan.engineering.azuki.graphs.adapter.api.GraphFunctionalElements
+import com.anaplan.engineering.azuki.graphs.dsl.GraphScenario
+import com.anaplan.engineering.azuki.graphs.graphA
+
+
+@BEH(GraphBehaviours.CreateDirectedGraph, GraphFunctionalElements.DirectedGraph, """
+    Create a directed graph
+""")
+class BEH8 : GraphScenario() {
+
+    @Eac("When a graph is first initialised it contains no edges or vertices")
```

Comment:
⛏️ I would say _created_ rather than _initialised_, since the AC is about a graph creation behaviour. So something like:
> When a graph is created, it contains no edges nor vertices